.github/workflows/release.yml

```diff
@@ -61,5 +61,7 @@ jobs:
             -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
             https://api.github.com/repos/google/closure-compiler-npm/releases \
             -d '{"tag_name":"v${{ github.event.inputs.COMPILER_VERSION_NUMBER }}.0.0","name":"${{ github.event.inputs.COMPILER_VERSION_NUMBER }}.0.0","body":"Closure-compiler ${{ github.event.inputs.COMPILER_VERSION_NUMBER }} release","draft":false,"prerelease":false,"generate_release_notes":true}'
-      - name: Trigger Build and Publish
-        uses: ./.github/workflows/build.yml
+  trigger-publication:
```

Comment:
So I interpret this change as something like the description below.
How far off am I?

The create-release workflow failed with an error message:
`Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/closure-compiler-npm/closure-compiler-npm/.github/workflows/build.yml'. Did you forget to run actions/checkout before running your local action?`

This was because ... something something... so the final step didn't think that file existed, even though it did.

This change creates a separate workflow specifically for triggering publication,
instead of doing it as a final step of the `create-release` workflow.

This allows us to add an explicit dependency on the `create-release` workflow and especially on the
`./.github/workflows/build.yml` file it creates.

Reply:
I had to go re-read the documentation on this: https://docs.github.com/en/actions/using-workflows/reusing-workflows

When I looked very carefully, these had to be set up as 2 separate jobs in the same workflow. I think the previous version I had just misread it.